srcpkgs/afew/template

```diff
@@ -1,25 +1,25 @@
 # Template file for 'afew'
 pkgname=afew
-version=1.3.0
+version=2.0.0
 revision=1
 archs=noarch
 build_style=python3-module
 pycompile_module="afew"
-hostmakedepends="git python3-setuptools python3-Sphinx pkg-config"
-depends="python3-setuptools notmuch-python3 python3-dkimpy"
```

Comment:
python3-setuptools is needed, and also python3-chardet

Reply:
python3-chardet is required but I don't think afew requires python3-setuptools.

I stopped using afew for a while, though

Reply:
It seems like afew doesn't require setuptools directly,
afew requires chardet and chardet requires setuptools.